pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -263,15 +275,15 @@ func (trs *TaskRunStatus) SetCondition(newCond *apis.Condition) {
 type StepState struct {
 	corev1.ContainerState `json:",inline"`
 	Name                  string `json:"name,omitempty"`
-	ContainerName         string `json:"container,omitempty"`
+	Container             string `json:"container,omitempty"`
 	ImageID               string `json:"imageID,omitempty"`
 }
 
 // SidecarState reports the results of running a sidecar in a Task.
 type SidecarState struct {
 	corev1.ContainerState `json:",inline"`
 	Name                  string `json:"name,omitempty"`
-	ContainerName         string `json:"container,omitempty"`
+	Container             string `json:"container,omitempty"`
```

Comment:
I believe this should also not be renamed as discussed in https://github.com/tektoncd/pipeline/pull/4402#discussion_r787932214

Reply:
Done.